frontend/packages/console-dynamic-plugin-sdk/CHANGELOG-core.md

```diff
@@ -1,7 +1,18 @@
 # Changelog for `@openshift-console/dynamic-plugin-sdk`
 
-Refer to [Console dynamic plugins README](./README.md) for OpenShift Console version vs SDK package
-version and PatternFly version compatibility.
+Console plugin SDK packages follow a semver scheme where the major and minor version number indicates
+the supported OCP Console version, and the patch version number indicates the release of that particular
+package. For example, package version `4.19.0` is the initial package release which supports Console 4.19.
+
+For 1.x plugin SDK packages, refer to "OpenShift Console Versions vs SDK Versions" compatibility table
+in [Console dynamic plugins README](./README.md).
+
+## 4.19.0 - 2025-02-13
```

Comment:
Do we want to use a `-prerelease` tag until we GA? @vojtechszocs Opinion?

Reply:
OK, will adapt to using prerelease tag in package version if the given Console version is not GA yet.